### ISSUE TITLE
Option to change BG color?

### ISSUE THREAD

**Issue by edgonzalez32** [utom/sketch-measure#519]
Because of accessibility needs, the white on green for specs fails. Can we have an option to change that color?

**Comment by deepesh55** [comment]
Yes please. Its really tough to read them. Do give us an option to change the color
